### ISSUE TITLE
[feature]: Promote Cycles to the Workspace level rather than Project level

### ISSUE THREAD

**Issue by Huuuze** [makeplane/plane#3337]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

Cycles are intended to manage tasking in a manner similar to Agile "sprints". As such, a [matrixed development team](https://www.youtube.com/watch?v=pPE9F34KPeI) may be assigned to work on multiple issues from multiple projects. The current implementation requires a Scrum master or project manager to move between multiple projects to get an individual's status on their tasking. 

This process is frustrating and a shortcoming for Plane.

Fortunately, it can be remedied. If Cycles were promoted to the Workspace level, a project manager could assign issues to a Cycle from any project in the Workspace. The project manager would also be able to use a single Agile board to move tasking through the workflow.

### Why should this be worked on?

The current implementation for Cycles requires project managers to navigate multiple projects to get status updates on issues.

**Comment by techspawn** [comment]
I agree with the points raised here. Currently, Cycles doesn't give the overall picture to project managers and teams working on different projects. They need to look into I think promoting Cycles to the Workspace level or providing radmap option where we can add tasks or cycles would make things a lot easier. Looking forward to seeing this improvement!

**Comment by etaxi341** [comment]
@srinivaspendem is that something that will be added to the roadmap of the plane development? Cycles is such a nice feature but really not usable when developers work on different projects

**Comment by tvarsis** [comment]
This should be the same for Modules. Very often an Epic/Module etc will require features from different projects, such as API/DB/Frontend etc. Not being able to tie features from different project to the same Module makes it lose the overview and relationship aspect of it.

**Comment by leedsjb** [comment]
Adding a +1 for this feature - or an alternative solution that solves the problem. We structure our Plane Projects by software product we are building. Our projects are worked on by multiple teams. For example, a frontend team building UI for a feature and a database team modifying the database for the same feature. For a team to plan their cycle they have to navigate between several per project cycles when cycles should be team based, not project based. An alternative to making cycles workspace level would be to make them teamspace level.

**Comment by leedsjb** [comment]
Similar to https://github.com/makeplane/plane/issues/5325 which requests cycles at the teamspace level.
